PerfTools/JeProf/plugins/JeProfService.h

```diff
@@ -0,0 +1,91 @@
+#ifndef IgTools_JeProf_JeProfService_h
+#define IgTools_JeProf_JeProfService_h
```

Comment:
Could you move the contents of this header to the `.cc` file?

Reply:
OK. I was just following the pattern or IgProfService.